### ISSUE TITLE
conflicting subparser: downsample in 3.0.0-beta

### ISSUE THREAD

**Issue by wlugmayr** [ml-struct-bio/cryodrgn#306]
hi,

whatever options i give to the main script in the new conda installation of this github code it issues the error message below.
means also for 
cryodrgn help
cryodrgn downsample
cryodrgn downsample -help
:

$ cryodrgn
Traceback (most recent call last):
  File "/gpfs/cssb/software/centos7/anaconda3/envs/cryodrgn-3.0.0b/bin/cryodrgn", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/gpfs/cssb/software/centos7/anaconda3/envs/cryodrgn-3.0.0b/cryodrgn/cryodrgn/__main__.py", line 65, in main
    this_parser = subparsers.add_parser(
                  ^^^^^^^^^^^^^^^^^^^^^^
  File "/gpfs/cssb/software/centos7/anaconda3/envs/cryodrgn-3.0.0b/lib/python3.11/argparse.py", line 1192, in add_parser
    raise ArgumentError(self, _('conflicting subparser: %s') % name)
argparse.ArgumentError: argument {downsample,preprocess}: conflicting subparser: downsample

**Comment by zhonge** [comment]
Hi, I am not sure I can reproduce this problem. How did you install the latest version? Can you try with a clean install of cryodrgn version 3.0.0-beta?

```
(cryodrgn) [Wed Oct 18 17:51 share] cryodrgn downsample -h
usage: cryodrgn downsample [-h] -D D -o MRCS [-b B] [--is-vol] [--chunk CHUNK] [--datadir DATADIR]
                           [--max-threads MAX_THREADS] [--ind PKL]
                           mrcs

Downsample an image stack or volume by clipping fourier frequencies

positional arguments:
  mrcs                  Input particles or volume (.mrc, .mrcs, .star, .cs, or .txt)

optional arguments:
  -h, --help            show this help message and exit
  -D D                  New box size in pixels, must be even
  -o MRCS               Output projection stack (.mrcs)
  -b B                  Batch size for processing images (default: 5000)
  --is-vol              Flag if input .mrc is a volume
  --chunk CHUNK         Chunksize (in # of images) to split particle stack when saving
  --datadir DATADIR     Optionally provide path to input .mrcs if loading from a .star or .cs file
  --max-threads MAX_THREADS
                        Maximum number of CPU cores for parallelization (default: 16)
  --ind PKL             Filter particle stack by these indices
  ```

**Comment by wlugmayr** [comment]
Hi, sorry my fault. I rechecked my installation and I used python=3 which gave me python 3.11 instead of 3.9. With python 3.9 install the error is gone.

**Comment by zhonge** [comment]
Thanks for the update. We can check if there is an incompatibility with python 3.11 version or if i was something else in conda...

**Comment by michal-g** [comment]
I was able to replicate this using Python 3.11 and `v3.1.0-b`. A big difficulty with us supporting 3.11 at the moment is that it requires using `numpy>=1.24`, whereas cryoDRGN only supports up to `v1.22` due to other known issues (see e.g. #318).

We should update `pyproject.toml` to ensure that either 3.9 or 3.10 are installed instead, and investigate solving the other underlying issues that prevent compatibility with 3.11. Closing this issue for now but keeping this general problem on our radar!
